DeviceAdapters/DahengGalaxy/ClassGalaxy.cpp

```diff
@@ -144,19 +144,19 @@ int ClassGalaxy::Initialize()
 
 
         vectorDeviceInfo.clear();
-        //ö���豸
```

Comment:
hmmm... these unicode issues seem to have been unintentionally made simply by saving the file (in VSCode).   @marktsuchida, do you know how i can avoid this?

Reply:
#469, in progress.

Reply:
#469 has been merged.

Reply:
k updated here as well